jlm/hls/backend/rvsdg2rhls/mem-conv.cpp

```diff
@@ -45,9 +46,9 @@ jlm::hls::route_request(rvsdg::Region * target, jlm::rvsdg::output * request)
   }
   else
   {
-    auto ln = dynamic_cast<jlm::hls::loop_node *>(request->region()->node());
-    JLM_ASSERT(ln);
-    auto output = jlm::rvsdg::structural_output::create(ln, request->Type());
+    auto lambda = dynamic_cast<jlm::hls::loop_node *>(request->region()->node());
+    JLM_ASSERT(lambda);
```

Comment:
The same here: `lambda` and `AssertedCast<>()`

Reply:
Same as above